brickpi/brickpi_ports_out.c

```diff
@@ -193,9 +193,9 @@ static unsigned brickpi_out_port_get_commands(void *context)
 		| BIT(TM_COMMAND_RESET);
 }
 
-static int brickpi_out_port_send_command(void *context,
-					 struct tacho_motor_params *params,
-					 enum tacho_motor_command command)
+static int brickpi_out_port_tacho_set_command(void *context,
```

Comment:
I'll check on this tomorrow - I had a good reason at the time but may need to revert it now